OCI-CLI_reference.md

```diff
@@ -0,0 +1,1218 @@
+# Introduction
+
+The *CLI* is a small-footprint tool that you can use on its own or with the Console to complete Oracle Cloud Infrastructure tasks. The CLI provides the same core functionality as the Console, plus additional commands. Some of these extend Console functionality, such as the ability to run scripts.  
```

Comment:
Think we should spell out "Command Line Interface (CLI)" or "CLI (Command Line Interface)" for the first usage? I prefer the former version (initialism in parens), myself.

Reply:
I pulled this definition from a page on the Oracle site, so my first inclination was to leave it intact. :) But I agree we should present the full form.